_posts/2016-02-03-ec2-provisioning-using-foreman-update.md

```diff
@@ -13,6 +13,9 @@ tags:
 - virtualizaition
 guest_post: true
 guest_url: http://zipkid.eu/
+excerpt: |
+  Back in 2012 Ohad Levy wrote an excellent [blog post](http://theforeman.org/2012/05/ec2-provisioning-using-foreman.html) on this subject. The concepts and steps to perform he explains are still mainly correct and relevant but the UI has changed enough to justify a new post. A lot of time has passed since then and many Foreman versions, with many improvements and changes, have passed.
```

Comment:
The Markdown link will need removing or replacing with HTML I think, it doesn't render.